<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShrinkAnimationClass</class>
 <widget class="QWidget" name="ShrinkAnimationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShrinkAnimation</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleWidget" name="upWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>32</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>0</y>
         <width>206</width>
         <height>32</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
	font: 12pt &quot;微软雅黑&quot;;
	color:#ffffff;
    background-color: blue;
}
QPushButton:pressed {
    font: 12pt &quot;微软雅黑&quot;;
	color:#ffffff;
    background-color: blue;
}</string>
       </property>
       <property name="text">
        <string>效果</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true"> QWidget#scrollAreaWidgetContents{
	background-color: rgb(1, 255, 43);
 }

 QScrollBar:vertical {
     background: #ffffff;
     width: 6px;
 }
 QScrollBar::handle:vertical {
     background:#cccccc;
     border-radius:3px;
     width: 6px;
 }

 QScrollBar::sub-line:vertical {
     background:transparent;
 }
 QScrollBar::add-line:vertical {
     background:transparent;
 }


 QScrollBar:horizontal {
     background: #ffffff;
     height: 6px;
 }
 QScrollBar::handle:horizontal {
     background:#cccccc;
     border-radius:3px;
     height: 6px;
 }

 QScrollBar::sub-line:horizontal {
     background:transparent;
 }
 QScrollBar::add-line:horizontal {
     background:transparent;
 }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>596</width>
           <height>399</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="shrinkanimation.qrc"/>
 </resources>
 <connections/>
</ui>
